src/main/java/sigma/command/Commands.java

```diff
@@ -0,0 +1,36 @@
+package sigma.command;
+
+import sigma.Storage;
+import sigma.TaskList;
+import sigma.Ui;
+import sigma.exception.SigmaException;
+
+/**
+ * Represents the command to execute the user's input.
+ */
+public abstract class Commands {
+
+    protected String[] split;
+    public Commands(String[] split) {
+        this.split = split;
+    }
+
+    /**
+     * Executes the command.
+     * @param tasks
+     * @param ui
+     * @param storage
+     * @throws SigmaException
+     */
+    public abstract void execute(TaskList tasks, Ui ui, Storage storage) throws SigmaException;
```

Comment:
This violates the Java coding standard, gotta wrap into a newline. See https://se-education.org/guides/conventions/java/intermediate.html#layout

Reply:
Isn't it for lines more than 120 characters? I have about 95 here.